addons/containerd/1.3.7/install.sh

```diff
@@ -3,13 +3,16 @@
 function containerd_install() {
     local src="$DIR/addons/containerd/1.3.7"
 
-    if [ "$SKIP_CONTAINERD_INSTALL" = "1" ]; then
-        return 0
+    if [ "$SKIP_CONTAINERD_INSTALL" != "1" ]; then
+        containerd_binaries "$src"
+        containerd_configure
+        containerd_service "$src"
     fi
 
-    containerd_binaries "$src"
-    containerd_configure
-    containerd_service "$src"
+    # NOTE: this will not remove the proxy
```

Comment:
I copied the logic here from docker_configure_proxy. Is this desired or should we remove the proxy when not configured?